### PR TITLE
[UPDATE BONUS GUIDE] Update joinmarket.md

### PR DESCRIPTION
**Pull request by Mynima**
#### What

New version of JoinMarket available. Updated to latest version and added $VERSION variable. Minor corrections.

#### Why

Simplify future maintenance, bring up to date with latest release. Minor bug fixes and suggested updates from user.

#### How

Quick replacement of old version throughout using `$ VERSION="0.9.9"`. Reduced the number of future updates to 2 instances (variable assignment and signature verification).

#### Scope

- [ ] significant change to core configuration
- [ ] independent bonus guide
- [x] simple bug fix

Fixes # (link issue)

#### Test & maintenance

Can easily be tested with a fresh installation of JoinMarket or upgrade to existing installation.

Look ma'

image

### CODE REVIEW

File: guide/bonus/bitcoin/joinmarket.md [hunk 1]
```diff
@@ -97,10 +97,11 @@ If you get `E: Package 'python-virtualenv' has no installation candidate` error
 * As user "joinmarket", download the latest release, signature and timestamp. First check for the latest release on the [Releases page](https://github.com/JoinMarket-Org/joinmarket-clientserver/releases) and update version numbers as you go if needed.
 
   ```sh
+  $ VERSION="0.9.9"
   $ cd /tmp
-  $ wget -O joinmarket-clientserver-0.9.8.tar.gz https://github.com/JoinMarket-Org/joinmarket-clientserver/archive/v0.9.8.tar.gz
-  $ wget https://github.com/JoinMarket-Org/joinmarket-clientserver/releases/download/v0.9.8/joinmarket-clientserver-0.9.8.tar.gz.asc
-  $ wget https://github.com/JoinMarket-Org/joinmarket-clientserver/releases/download/v0.9.8/joinmarket-clientserver-0.9.8.tar.gz.asc.ots
+  $ wget -O joinmarket-clientserver-v$VERSION.tar.gz https://github.com/JoinMarket-Org/joinmarket-clientserver/archive/v$VERSION.tar.gz
```
Comment (kristapsk) [inline guide/bonus/bitcoin/joinmarket.md:102]:
```suggestion
  $ wget -O joinmarket-clientserver-$VERSION.tar.gz https://github.com/JoinMarket-Org/joinmarket-clientserver/archive/v$VERSION.tar.gz
```

**Review by kristapsk — APPROVED** [review]

File: guide/bonus/bitcoin/joinmarket.md [hunk 2]
```diff
@@ -412,11 +413,11 @@ If upgrading from pre-0.8.1 to a newer versions, note that yield generator confi
...
 
-* Download, verify, extract and install the JoinMarket as described in the [Install JoinMarket](#install-joinmarket) section of this guide.
+* Download, verify, extract and install the JoinMarket as described in the [Install JoinMarket](joinmarket.md#installation) section of this guide.
 
 * Optionally delete old JoinMarket version directory to free up about 200 megabytes.
   
   ```sh
   $ cd
-  $ rm -rf joinmarket-clientserver-0.9.8
+  $ rm -rf joinmarket-clientserver-$VERSION
```
Comment (sig-noi) [inline guide/bonus/bitcoin/joinmarket.md:422]: Would this not delete the new version of JoinMarket?

File: guide/bonus/bitcoin/joinmarket.md [hunk 3]
```diff
@@ -412,11 +413,11 @@ If upgrading from pre-0.8.1 to a newer versions, note that yield generator confi
   $ unlink joinmarket
   ```
 
-* Download, verify, extract and install the JoinMarket as described in the [Install JoinMarket](#install-joinmarket) section of this guide.
+* Download, verify, extract and install the JoinMarket as described in the [Install JoinMarket](joinmarket.md#installation) section of this guide.
```
Comment (sig-noi) [inline guide/bonus/bitcoin/joinmarket.md:417]: * Move the existing joinmarket.cfg file and generate a new joinmarket.cfg, then customize it per the [Configuration](joinmarket.md#configuration) section of this guide.

```sh
$ cd
$ mv /data/joinmarket/joinmarket.cfg /data/joinmarket/joinmarket.cfg.old
$ . /activate.sh
$ (jvmenv) $ ./wallet-tool.py 
```

File: guide/bonus/bitcoin/joinmarket.md (hunk 2 above)
Comment (kristapsk) [reply guide/bonus/bitcoin/joinmarket.md:422]: Nice catch! Didn't test this one. Should be changed.

Comment (Mynima) [reply guide/bonus/bitcoin/joinmarket.md:422]: Indeed, good catch!

File: guide/bonus/bitcoin/joinmarket.md (hunk 3 above)
Comment (Mynima) [reply guide/bonus/bitcoin/joinmarket.md:427]: Added, please let me know if you have any additional comments.

**Review by kristapsk — APPROVED** [review]

**Review by sig-noi — APPROVED** [review]
ACK. Already upgraded so didn’t fully test.

**Pull request merged**